Assets/MRTK/SDK/Experimental/PinchSliderBox/PinchSliderBox.cs

```diff
@@ -0,0 +1,499 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
nit: it's been so long, we can update these license headers to the new style

Reply:
Updated!